spec/integ/debug-api.spec.js

```diff
@@ -24,7 +24,7 @@ describe("Debug API", () => {
     it("should enable the debug API", async () => {
         // Revert to the default value for this one test
         expect(env.ircBridge.debugApi).not.toBeNull();
-        const res = await request(`http://localhost:${DEBUG_PORT}/version?access_token=${asToken}`);
+        const res = await (await crossFetch(`http://localhost:${DEBUG_PORT}/version?access_token=${asToken}`)).text();
```

Comment:
Just removing a dependency on the legacy request. Crossfetch is my fave since it's just `fetch` and soon enough Node will have that.